metricbeat/module/system/process/_meta/fields.yml

```
@@ -693,6 +693,49 @@
               description: >
                 Memory that cannot be reclaimed, in bytes.
 
+            - name: pressure
+              type: group
+              description: Pressure (resource contention) stats.
+              fields:
+                - name: some
+                  type: group
+                  description: Share of time in which at least some tasks are stalled on a given resource
+                  fields:
+                    - name: 10.pct
+                      type: float
+                      format: percent
+                      description: Pressure over 10 seconds
+                    - name: 60.pct
+                      type: float
+                      format: percent
+                      description: Pressure over 60 seconds
+                    - name: 300.pct
+                      type: float
+                      format: percent
+                      description: Pressure over 300 seconds
+                    - name: total
+                      type: long
+                      description: total Some pressure time
+                - name: full
+                  type: group
+                  description: Share of time in which all non-idle tasks are stalled on a given resource simultaneously
+                  fields:
+                    - name: 10.pct
+                      type: float
+                      format: percent
+                      description: Pressure over 10 seconds
+                    - name: 60.pct
+                      type: float
+                      format: percent
+                      description: Pressure over 60 seconds
+                    - name: 300.pct
+                      type: float
+                      format: percent
+                      description: Pressure over 300 seconds
+                    - name: total
+                      type: long
+                      description: total Full pressure time
```

Would this be accurate?

```suggestion
                      description: Total time that all non-idle tasks are stalled
```